OverlayPlugin.Core/MemoryProcessors/Combatant/CombatantMemory.cs

```diff
@@ -150,6 +150,11 @@ public unsafe List<Combatant> GetCombatantList()
         // This function always returns a combatant object, even if empty.
         protected abstract unsafe Combatant GetCombatantFromByteArray(byte[] source, uint mycharID, bool isPlayer, bool exceptEffects = false);
 
+        protected unsafe byte[] GetByteArray(IntPtr address, int length)
+        {
+            return memory.GetByteArray(address, length);
+        }
+
```

Comment:
This is just a blind wrapper around the FFXIVMemory class, I think it would be better to just pull the reference to the class into CombatantMemory63 instead.

Reply:
Removed this and adjusted the visibility of this reference to `protected` - if this is not what you meant by your comment you'll have to hold my hand a bit on the resolution 😅

Reply:
> you'll have to hold my hand a bit on the resolution

1. Remove all changes to `CombatantMemory.cs`
2. Add `private FFXIVMemory memory;` to `CombatantMemory63.cs`
3. In `CombatantMemory63`'s constructor, `memory = container.Resolve<FFXIVMemory>();`
4. Use the local reference to `FFXIVMemory` instead